api/libproject/devstart.rb

```diff
@@ -1589,38 +1589,56 @@ def export_workspace_data(cmd_name, *args)
     :fn => ->(*args) {export_workspace_data("export-workspace-data", *args)}
 })
 
-def generate_impersonated_user_token(cmd_name, *args)
+def generate_impersonated_user_tokens(cmd_name, *args)
   common = Common.new
   ensure_docker cmd_name, args
 
   op = WbOptionsParser.new(cmd_name, args)
   op.add_typed_option(
-      "--output-token-filename [output-token-filename]",
+      "--output-token-filenames [output-token-filename1, ...]",
       String,
-      ->(opts, v) { opts.output_token_filename = v},
-      "Path to an output file for the generated token")
+      ->(opts, v) { opts.output_token_filenames = v},
+      "Comma-separated paths to output file(s) for the generated token(s)")
```

Comment:
nit: would be good to include in one or both of these comments that the order should correspond to that of --impersonated_usernames